events/babylon_discord_break_2_1_event.txt

```diff
@@ -0,0 +1,321 @@
+﻿namespace = ideology_break_2_1_event
+# Ideological break for level 2 reached
+ideology_break_2_1_event.1 = {
+	#Secret society
+	type = country_event
+	title = ideology_break_2_1_event.1.t
+	desc = ideology_break_2_1_event.1.d
+	flavor = ideology_break_2_1_event.1.f
+	event_image = {
+		video = "gfx/event_pictures/africa_leader_arguing.bk2"
+	}
+	icon = "gfx/interface/icons/event_icons/event_fire.dds"
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	duration = 3
+	trigger = {
+		# Caused when a nation hits 200 on their cohesion level.
```

Comment:
chance to be caused, otherwise you would just put a trigger here.

Reply:
fixing it